### PR TITLE
Bootstrap upgrade

### PR DESCRIPTION
**Pull request by nimerb**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [X] Chart Version bumped
- [X] CHANGELOG.md updated
- [X] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
Since the bootstrap.creds is loaded only on startup, after applying the bootstap.creds for a running Artifactory, the pod of Artifactory will be restarted automatically to load the changes.

### CODE REVIEW

**Comment by rimusz** [comment]
@nimerb if this PR has #400 and #401 then close them

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.14.6] - Jun 27, 2019
+* Updated Artifactory version to 6.11.0
+
+## [0.14.5] - Jun 27, 2019
+* Restart Primary node when bootstrap.creds file has been modified in artifactory-ha
```
Comment (danielezer) [inline stable/artifactory-ha/CHANGELOG.md]: I’m pretty sure the 1.14.5 version will not be built, you should join the two changes into one version or use two PRs

Comment (nimerb) [reply stable/artifactory-ha/CHANGELOG.md]: merged two changes in one version

File: stable/artifactory/templates/artifactory-statefulset.yaml [hunk 2]
```diff
@@ -29,7 +29,8 @@ spec:
         release: {{ .Release.Name }}
       annotations:
         checksum/binarystore: {{ include (print $.Template.BasePath "/artifactory-binarystore.yaml") . | sha256sum }}
-      {{- range $key, $value := .Values.artifactory.annotations }}
+        checksum/access-creds: {{ include (print $.Template.BasePath "/access-bootstrap-creds.yaml") . | sha256sum }}
+              {{- range $key, $value := .Values.artifactory.annotations }}
```
Comment (danielezer) [inline stable/artifactory/templates/artifactory-statefulset.yaml]: Fix the indentation pls

File: stable/artifactory/templates/artifactory-statefulset.yaml [hunk 3]
```diff
@@ -29,6 +29,7 @@ spec:
         release: {{ .Release.Name }}
       annotations:
         checksum/binarystore: {{ include (print $.Template.BasePath "/artifactory-binarystore.yaml") . | sha256sum }}
+        checksum/access-creds: {{ include (print $.Template.BasePath "/access-bootstrap-creds.yaml") . | sha256sum }}
```
Comment (danielezer) [inline stable/artifactory/templates/artifactory-statefulset.yaml]: You need to add an if statement to check if no pre existing secret was provided. This file is not always used for the access bootstrap

File: stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml [hunk 4]
```diff
@@ -29,6 +29,7 @@ spec:
         release: {{ .Release.Name }}
       annotations:
         checksum/binarystore: {{ include (print $.Template.BasePath "/artifactory-binarystore.yaml") . | sha256sum }}
+        checksum/access-creds: {{ include (print $.Template.BasePath "/access-bootstrap-creds.yaml") . | sha256sum }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/artifactory-primary-statefulset.yaml]: Same here

**Pull request merged**